cmd/kube-scheduler/app/server.go

```diff
@@ -283,7 +283,7 @@ func buildHandlerChain(handler http.Handler, authn authenticator.Request, authz
 	handler = genericapifilters.WithAuthentication(handler, authn, failedHandler, nil, nil)
 	handler = genericapifilters.WithRequestInfo(handler, requestInfoResolver)
 	handler = genericapifilters.WithCacheControl(handler)
-	handler = genericfilters.WithHTTPLogging(handler)
+	handler = genericfilters.WithHTTPLogging(handler, func(_ context.Context, f func()) { f() })
```

Comment:
This is very hard to think about what's happening.

Why can't we just do the following:
- change `withLogging` to do more-or-less exactly what's `ListResource` is doing [check if TaskFrom is non-nil and if so append to it]
- not change anything else...

Reply:
> change withLogging to do more-or-less exactly what's ListResource is doing [check if TaskFrom is non-nil and if so append to it]

Actually my first attempt was like that. But it caused cycle import if we call TaskFrom in `httplog`. Let me try if I can work this around.

Reply:
Not sure if it's an overkill to move routine filter in a separate directory to tackle the cycle import issue. But it should be what you suggested.